src/Symfony/Component/Validator/Tests/Constraints/NotBlankValidatorTest.php

```diff
@@ -98,4 +98,33 @@ public function testEmptyArrayIsInvalid()
             ->setCode(NotBlank::IS_BLANK_ERROR)
             ->assertRaised();
     }
+
+    /**
+     * @dataProvider provideAllowNull
+     */
+    public function testAllowNull(bool $allowNull)
+    {
+        $constraint = new NotBlank(array(
+            'message' => 'myMessage',
+            'allowNull' => $allowNull,
+        ));
+
+        $this->validator->validate(null, $constraint);
+
+        if ($allowNull) {
```

Comment:
I think the test would be more readable when being split and given two meaningful names.